CMakeLists.txt

```diff
@@ -268,6 +275,9 @@ if(PORTABLE)
     if(CMAKE_SYSTEM_PROCESSOR MATCHES "^s390x")
       set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=z196")
     endif()
+    if(CMAKE_SYSTEM_PROCESSOR MATCHES "^loongarch64")
```

Comment:
I am not an expert in how this is supposed to work with PORTABLE, but would guess you need lines modeled on this:    
if(CMAKE_SYSTEM_PROCESSOR MATCHES "^s390x" AND NOT HAS_S390X_MARCH_NATIVE)
      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=z196")

around line 288

Reply:
i checked https://gcc.gnu.org/onlinedocs/gcc/LoongArch-Options.html#LoongArch-Options , but cannot find the document on the default "cpu-type" for `-march=cpu-type`.  the same applies to https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html and https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html#ARM-Options. guess it'd be safe to specify this option explicitly instead of relying on the default flags of gcc when it gets compiled?